website/docs/d/database_user.html.markdown

```diff
@@ -111,4 +112,4 @@ Containing key-value pairs that tag and categorize the database user. Each key a
 * `key` - The key that you want to write.
 * `value` - The value that you want to write.
 
-See [MongoDB Atlas API](https://docs.atlas.mongodb.com/reference/api/database-users-get-single-user/) Documentation for more information.
\ No newline at end of file
+See [MongoDB Atlas API](https://docs.atlas.mongodb.com/reference/api/database-users-get-single-user/) Documentation for more information.
```

Comment:
nit pick: although the redirect seems to be functioning properly, I'd prefer if we linked to the latest version of the API to cut down on the number of redirects
```suggestion
See [MongoDB Atlas API](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Database-Users/operation/getDatabaseUser) Documentation for more information.
```